ajax/libs/fast-xml-parser/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "fast-xml-parser",
+  "version": "2.2.1",
+  "description": "Parse XML to JS/JSON very fast without C/C++ based libraries",
+  "filename": "parser.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/NaturalIntelligence/fast-xml-parser"
+  },
+  "keywords": [
+    "fast",
+    "xml",
+    "json",
+    "parser",
+    "xml2js",
+    "xml2json",
+    "js",
+    "traversal",
+    "cli",
+    "command"
+  ],
+  "author": "Amit Gupta (https://github.com/amitguptagwl)",
+  "license": "MIT",
+  "npmName": "fast-xml-parser",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "parser*.(js|css|map)"
```

Comment:
@hare1039 Me either. I found all the directory are empty, but I don't know how to fix it. I've checked that the spelling is correct.